app/models/user_model.rb

```diff
@@ -0,0 +1,23 @@
+class UserModel
+  attr_accessor :id, :user, :status
+
+  def initialize(id, user, status)
+    @id = id
+    @user = user
+    @status = status
+  end
+
+  def user_exists?(csv_data)
+    csv_data.any? { |row| row['id'].to_i == @id.to_i }
```

Comment:
[suggestion]
Could you try to configure Activerecord to persist data to a Sqlite?